tools/build/README.md

```diff
@@ -1,11 +1,9 @@
 # Build helper scripts
 
 This directory contains the following utilities.
-- `scanCode.py`: checks all code for conformance with respect to certain conventions,
```

Comment:
since the python script is not remove yet, leave it here. But add a note about the new way of doing scancode and the new repo collection, and that the script is planned to be remove soon.